lib/api/list.js

```diff
@@ -43,6 +43,18 @@ List.prototype = {
 
   },
 
+  removeLeadsFromList: function(listId, leadIds) {
+    if (!_.isArray(leadIds) && !_.isEmpty(leadIds)) {
+      var msg = 'input must be an array of lead ids';
+      log.error(msg);
+      return Promise.reject(msg);
+    }
+
+    var path = util.createPath('lists', listId, 'leads.json');
+    leadIds = [].concat(leadIds); // ensure it's an array
+    return this._connection.del(path + "?id=" + leadIds.join('&id='), {headers: {"Content-Type": "application/json"}});
```

Comment:
Same with the other comment, can you see if it is possible to combine the query parameters with the headers into a single object?

```js
this._connection.del(path, {id: leadIds.join('&id='), headers: ...});
```

In addition, I've been using single quotes `'` in this code unless I'm quoting something that contains a single quote.